printer_mods/teookie/skr_brace/readme.md

```diff
@@ -1,7 +1,7 @@
 SKR Brace.
 
 Clips your SKR DIN rail mounts together
-- Rigidizes and imobolizes the mounts
+- Rigidizes and immobolizes the mounts
```

Comment:
That's still misspelled, should be `immobilizes`

Reply:
fixed for realz this time. Will spell check better in the future. Thanks.